morse_code.rb

```diff
@@ -0,0 +1,29 @@
+def decode_char(code)
+  decoder = { '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+              '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N',
+              '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U',
+              '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z', '.----' => '1',
+              '..---' => '2', '...--' => '3', '....-' => '4', '.....' => '5', '-....' => '6', '--...' => '7',
+              '---..' => '8', '----.' => '9', '-----' => '0' }
+  decoder[code]
+end
+
+puts decode_char('.-')
+
+def decode_word(code)
+  result = ''
+  arr = code.split
+  arr.each { |item| result += decode_char(item) }
+  result
+end
+
+puts decode_word('-- -.--')
+
+def decode(code)
+  result = ''
+  arr = code.split('   ')
+  arr.each { |item| result += " #{decode_word(item)}" }
+  result.strip
+end
```

Comment:
- [ ] Your application is not properly decoded. Kindly ensure to improve upon this. 

**Your decoded message:**
![image](https://user-images.githubusercontent.com/12745474/155578634-5139fe83-16ac-4dc8-ba5a-c50fd66e60f3.png)

**Expected:**
![image](https://user-images.githubusercontent.com/12745474/155578761-f7262f55-650c-4475-8345-2e9f6bcdd53f.png)